platform/android/detect.py

```diff
@@ -178,7 +178,7 @@ def mySpawn(sh, escape, cmd, args, env):
 
     env.Append(CPPFLAGS=["-isystem", sysroot + "/usr/include"])
     env.Append(CPPFLAGS=string.split('-fpic -ffunction-sections -funwind-tables -fstack-protector-strong -fvisibility=hidden -fno-strict-aliasing'))
-    env.Append(CPPFLAGS=string.split('-DANDROID -DNO_STATVFS -DGLES2_ENABLED'))
+    env.Append(CPPFLAGS=string.split('-DANDROID_ENABLED -DNO_STATVFS -DGLES2_ENABLED'))
```

Comment:
You're right! That made me think `ANDROID` was defined as well for a reason, but further research has shown even Google discourages its use. People wanting to check for Android would use `__ANDROID__` instead and it should be added by the compiler as long as the right ABI is specified.